action.yml

```diff
@@ -27,11 +27,53 @@ inputs:
     description: 'Apply custom chapters for PRs without linked issues.'
     required: false
     default: 'true'
+  verbose:
+    description: 'Print verbose logs.'
+    required: false
+    default: 'false'
+outputs:
+  release-notes:
+    description: 'Generated release notes.'
+    value: ${{ steps.release-notes-generator.outputs.release-notes }}
 
 branding:
   icon: 'book'
   color: 'yellow'
 
 runs:
-  using: 'node20'
-  main: 'dist/index.js'
+  using: 'composite'
+  steps:
+    - name: Install Python dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        python3 -m pip install -r ${{ github.action_path }}/requirements.txt
+      shell: bash
+
+    - name: Call command python3
+      run: python3 --version
+      shell: bash
+
+    - name: Set PROJECT_ROOT and update PYTHONPATH
+      run: |
+        ACTION_ROOT="${{ github.action_path }}"
+        export PYTHONPATH="${PYTHONPATH}:${ACTION_ROOT}/release_notes_generator"
+      shell: bash
+
+    - name: Call Release Notes Generator
+      id: release-notes-generator
+      env:
+        INPUT_GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
+        INPUT_TAG_NAME: ${{ inputs.tag-name }}
+        INPUT_CHAPTERS: ${{ inputs.chapters }}
+        INPUT_WARNINGS: ${{ inputs.warnings }}
+        INPUT_PUBLISHED_AT: ${{ inputs.published-at }}
+        INPUT_SKIP_RELEASE_NOTES_LABEL: ${{ inputs.skip-release-notes-label }}
+        INPUT_PRINT_EMPTY_CHAPTERS: ${{ inputs.print-empty-chapters }}
+        INPUT_CHAPTERS_TO_PR_WITHOUT_ISSUE: ${{ inputs.chapters-to-pr-without-issue }}
+        INPUT_VERBOSE: ${{ inputs.verbose }}
+
+        INPUT_GITHUB_REPOSITORY: ${{ github.repository }}
+
+      run: |
+        python3 ${{ github.action_path }}/main.py
```

Comment:
I would rather use just `python` instead of `python3` - re previous comments.

Reply:
Used and tested in commit 61a5cf87ea4f9083ef9ed14eed501c6f90e5a980.